backend/editor/models.py

```diff
@@ -16,3 +16,70 @@ class Header(Marginal):
 
 class Footer(Marginal):
     pass
+
+
+# Models for FastAPI
+
+
+class ImportFromGithubParameters(BaseModel):
+    branch: str
+    taxonomy_name: str
+
+
+class ImportFromGithubResponse(BaseModel):
+    status: str
+
+
+class CreateNodeParameters(BaseModel):
+    branch: str
+    taxonomy_name: str
+
+
+class EditEntryParameters(BaseModel):
+    branch: str
+    taxonomy_name: str
+    entry: str
+
+
+class EditEntryResponse(BaseModel):
+    result = []

```

Comment:
We want more descriptive information about result structure. As is, this does not gives a lot of information to the reader.

There again I think there is potential for factorization. We should have an Entry class describing what is returned in an entry.
And here: `result : list[Entry] = Query(description…)` (if I remember correctly the type of result we get)

Reply:
Would the `Entry` class have all the parameters from the `EditEntryParameters` class?  I am confused here. 


Also, shall I make similar changes to `EditChildrenResponse`, `EditSynonymResponse`, `EditHeaderResponse`, `EditFooterResponse`? Because I am returning empty `[]` there as `result` too.

Reply:
I have made 2 new commits though.

Reply:
ok, you are heading in the right direction :-)

No you should add more description (describe every parameter) and also more structure for Entry for example.
Look at requests to see the structure, for example: https://api.taxonomy.openfoodfacts.net/allergens/test/entry/en:lupin

Reply:
sorry @Jagrutiti I just corrected my last comment, because I wrote "No" instead of "Now" :see_no_evil: